client.go

```diff
@@ -247,6 +248,19 @@ func (c *Client) getErrorFromResponse(resp *http.Response) (*errorObject, error)
 	return &s, nil
 }
 
+// Helper function to determine wither additional parameters should use ? or & to append args
+func getBasePrefix(basePath string) string {
+	var conjunction string
+	// check to see if there are already arguments in basePath (look for ? in path)
+	re := regexp.MustCompile(".*/[^/]*\\?[^/]*$")
```

Comment:
This currently recompiles the entire regular expression for each method call that uses `pagedGet()`, at a minimum it should probably be a package-level unexported variable so that it can be reused.

Reply:
Obsolete due to other changes.